server/pkg/builtin/manifest.yml

```diff
@@ -2544,7 +2544,7 @@ extensions:
               title: Background Color
               ui: color
             - id: showLayers
-              type: string
+              type: map
```

Comment:
![image](https://github.com/reearth/reearth/assets/20891087/53e03ee9-72f9-40e5-b499-c37a29d6639a)
The design has been updated, can you add another property with same type as map - hideLayers?

Reply:
we'll add it in the next PR.